__tests__/app.test.js

```diff
@@ -132,3 +132,64 @@ describe("Sepcific comments request by review ID /api/reviews/:review_id/comment
       });
   });
 });
+
+describe("3. POST requests.", () => {
+  test("status 201 adds a new comment object to the comments table and returns the added object", () => {
+    const newComment = {
+      user_name: "mallionaire",
+      body: "What a game!",
+    };
+    return request(app)
+      .post("/api/reviews/4/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toEqual(
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            body: "What a game!",
+            review_id: 4,
+            author: "mallionaire",
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          })
+        );
+      });
+  });
+  test("status 400 when given an invalid user_name property", () => {
+    const newComment = {
+      user_name: "board_gamer_90",
+      body: "unwanted comment",
+    };
+    return request(app)
+      .post("/api/reviews/4/comments")
+      .send(newComment)
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad request. Try again...");
+      });
+  });
+  test("status 404 when given an invalid review_id property", () => {
+    const newComment = {
+      user_name: "mallionaire",
+      body: "unwanted comment",
+    };
+    return request(app)
+      .post("/api/reviews/30/comments")
+      .send(newComment)
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad request. Try again...");
+      });
+  });
+  test("status 400 when given an empty object body", () => {
+    const newComment = {};
+    return request(app)
+      .post("/api/reviews/3/comments")
+      .send(newComment)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Please input information");
```

Comment:
might be better as `please input all fields`
I presume you get the same message even if there's one field, instead of two.